src/main.js

```diff
@@ -2,9 +2,23 @@
 import React from 'react';
 import ReactDom from 'react-dom';
 import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
-import ReduxConter from './components/counter';
+import ReduxCounter from './components/counter';
+import Metronome from './utils/metronome';
 
-const App = () => (
+class App extends React.Component {
```

Comment:
this file (`src/main.js`) is not included in the test coverage report. Apparently, the report covers only files inside at least second level folders. This file has no tests